HW7Class8/Task2.py

```diff
@@ -0,0 +1,35 @@
+# T ask 2
+# Creating a dictionary.
+# Create a function called make_country, which takes in a country’s name and capital
+# as parameters. Then create a dictionary from those two, with ‘name’
+# as a key and ‘capital’ as a parameter. Make the function print out the values of
+# the dictionary to make sure that it works as intended.
+
+from countryinfo import CountryInfo
```

Comment:
it is not build-in module. Please specify library name and its version in the comment to the task